src/parser/jobs/brd/modules/BurstWindow.tsx

```diff
@@ -0,0 +1,449 @@
+			</Trans>
+		</Message.Content>
+	</Message>
+
+	override initialise() {
+		super.initialise()
+
+		this.ignoreActions(SUPPORT_ACTIONS.map(actionKey => this.data.actions[actionKey].id))
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		const buffFilter = playerFilter.status(this.data.statuses.ARMYS_MUSE.id)
+		this.addEventHook(buffFilter.type('statusApply'), this.onApplyMuse)
+		this.addEventHook(buffFilter.type('statusRemove'), this.onRemoveMuse)
+		this.addEventHook(playerFilter.status(this.data.statuses.BARRAGE.id).type('statusRemove'), this.onRemoveBarrage)
+
+		const suggestionWindowName = <Trans id="brd.burst.suggestion.burstwindow">Burst Window</Trans>
+
+		this.addEvaluator(new RequiredGcdCountEvaluator({
+			requiredGcds: 7,
+			hasStacks: false,
+			globalCooldown: this.globalCooldown,
+			suggestionIcon: this.data.actions.RAGING_STRIKES.icon,
+			suggestionContent: <Trans id="brd.burst.suggestions.missedgcd.content">
+				Try to land 7 GCDs (8 GCDs with <DataLink status="ARMYS_MUSE"/>) during every {suggestionWindowName}.
+			</Trans>,
+			suggestionWindowName,
+			severityTiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			adjustCount: this.adjustMuseGcdCount.bind(this),
+			isRushed: this.isRushedEndOfPullWindow.bind(this),
+		}))
+
+		this.addEvaluator(new BarrageIronJawsEvaluator({
+			expectedActions: [
+				{
+					action: this.data.actions.BARRAGE,
+					expectedPerWindow: 1,
+				},
+				{
+					action: this.data.actions.IRON_JAWS,
+					expectedPerWindow: 1,
+				},
+			],
+			suggestionIcon: this.data.actions.BARRAGE.icon,
+			suggestionContent: <Trans id="brd.burst.suggestions.trackedactions.content">
+				One use of <DataLink action="BARRAGE"/> and one use of <DataLink action="IRON_JAWS"/> should occur during every {suggestionWindowName}.
+			</Trans>,
+			suggestionWindowName,
+			severityTiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			adjustCount: this.adjustExpectedBarrageIronJawsCount.bind(this),
+			adjustOutcome: this.adjustExpectedIronJawsOutcome.bind(this),
+			barrageId: this.data.actions.BARRAGE.id,
+			wasBarrageUsed: this.wasBarrageUsed.bind(this),
+		}))
+
+		this.addEvaluator(new ExpectedActionsEvaluator({
+			expectedActions: [
+				{
+					action: this.data.actions.APEX_ARROW,
+					expectedPerWindow: 1,
+				},
+				{
+					action: this.data.actions.BLAST_ARROW,
+					expectedPerWindow: 1,
+				},
+			],
+			suggestionIcon: this.data.actions.BLAST_ARROW.icon,
+			suggestionContent: <Trans id="brd.burst.suggestions.aaba-evaluator.content">
+				One use of <DataLink action="APEX_ARROW"/> and <DataLink action="BLAST_ARROW"/> should occur during every {suggestionWindowName} after the opener. Make sure you have at least 80 Soul Voice Gauge for your buffs.
+			</Trans>,
+			suggestionWindowName,
```

Comment:
I think `suggestionWindowName` should just be "Burst", otherwise the default suggestion text is going to say something like "... actions were missed during Burst Window windows." You might need to refactor other places where this variable is being used.